params_test.go

```diff
@@ -216,20 +216,23 @@ func TestListParams_Expand(t *testing.T) {
 func TestListParams_ToParams(t *testing.T) {
 	listParams := &stripe.ListParams{
 		Context:       context.Background(),
-		StripeAccount: TestMerchantID,
+		StripeAccount: stripe.String(TestMerchantID),
```

Comment:
Shouldn't we use `params.SetStripeAccount()` instead here?

Reply:
Yeah, I guess it can't hurt just to demonstrate our suggested usage. I'll change it.

Reply:
Pushed in 4e88ccfc4ed567c8b0776e4c6200e5a667f5c44f.